_search-plugins/search-pipelines/index.md

```diff
@@ -0,0 +1,387 @@
+
+For convenience, you can set a default search pipeline on an index. Once your index has a default pipeline, you don't need to specify the `search_pipeline` query parameter with every search request.
+
+### Setting a default search pipeline on an index
+
+To set a default search pipeline on an index, specify the `index.search.default_pipeline` in the index's settings:
+
+```json
+PUT /my_index/_settings 
+{
+  "index.search.default_pipeline" : "my_pipeline"
+}
+```
+{% include copy-curl.html %}
+
+After setting the default pipeline on `my_index`, you can try the same search for all documents:
+
+```json
+GET /my_index/_search
+```
+{% include copy-curl.html %}
+
+The response contains only the public document, indicating that the pipeline was applied by default:
+
+<details open markdown="block">
+  <summary>
+    Response
+  </summary>
+  {: .text-delta}
+
+```json
+{
+  "took" : 19,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 1,
+      "relation" : "eq"
+    },
+    "max_score" : 0.0,
+    "hits" : [
+      {
+        "_index" : "my_index",
+        "_id" : "1",
+        "_score" : 0.0,
+        "_source" : {
+          "message" : "This is a public message",
+          "visibility" : "public"
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+### Disabling the default pipeline for a request
+
+If you want to run a search request without applying the default pipeline, you can set the `search_pipeline` query parameter to `_none`:
+
+```json
+GET /my_index/_search?search_pipeline=_none
+```
+{% include copy-curl.html %}
+
+### Removing the default pipeline
+
+To remove the default pipeline from an index, set it to `null` or `_none`:
+
+```json
+PUT /my_index/_settings 
+{
+  "index.search.default_pipeline" : null
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT /my_index/_settings 
+{
+  "index.search.default_pipeline" : "_none"
+}
+```
+{% include copy-curl.html %}
+
+## Updating a search pipeline
+
+To update a search pipeline dynamically, replace the search pipeline using the Search Pipeline API. 
+
+#### Example request
+
+The following request upserts `my_pipeline` by adding a `filter_query` request processor and a `rename_field` response processor:
+
+```json
+PUT /_search/pipeline/my_pipeline
+{
+  "request_processors": [
+    {
+      "filter_query": {
+        "tag": "tag1",
+        "description": "This processor returns only publicly visible documents",
+        "query": {
+          "term": {
+            "visibility": "public"
+          }
+        }
+      }
+    }
+  ],
+  "response_processors": [
+    {
+      "rename_field": {
+        "field": "message",
+        "target_field": "notification"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+## Search pipeline versions
+
+You can specify a version for your pipeline when creating it in the `version` parameter:
```

Comment:
Do we mean "When creating your pipeline, you can specify a version for it in the `version` parameter"?